src/backdrop-blur/BackdropBlurFilter.ts

```diff
@@ -0,0 +1,135 @@
+import {
+    BlurFilter,
+    BlurFilterOptions,
+    Filter,
+    FilterSystem,
+    GlProgram,
+    GpuProgram,
+    RenderSurface,
+    Texture,
+    TexturePool,
+} from 'pixi.js';
+import { vertex, wgslVertex } from '../defaults';
+
+export type BackdropBlurFilterOptions = BlurFilterOptions;
+
+/**
+ * The BackdropBlurFilter applies a Gaussian blur to everything behind an object, and then draws the object on top of it.
+ *
+ * @class
+ * @extends BlurFilter
+ * @see {@link https://www.npmjs.com/package/pixi-filters|pixi-filters}
+ */
+export class BackdropBlurFilter extends BlurFilter
+{
+    private _blendPass: Filter;
+
+    /**
+     * @param options - The options of the blur filter.
+     * @param options.strength - The strength of the blur filter.
+     * @param options.quality - The quality of the blur filter.
+     * @param options.kernelSize - The kernelSize of the blur filter.Options: 5, 7, 9, 11, 13, 15.
+     */
+    constructor(options?: BackdropBlurFilterOptions);
+    /** @deprecated since 8.0.0 */
+    constructor(strength?: number, quality?: number, resolution?: number, kernelSize?: number);
+    constructor(...args: [BackdropBlurFilterOptions?] | [number?, number?, number?, number?])
+    {
+        super(...(args as ConstructorParameters<typeof BlurFilter>));
+
+        this.blendRequired = true;
+        this.padding = 0;
+
+        this._blendPass = new Filter({
+            gpuProgram: GpuProgram.from({
+                vertex: {
+                    source: wgslVertex,
+                    entryPoint: 'mainVertex',
+                },
+                fragment: {
```

Comment:
Could you follow the pattern like the other filters and externalize the shaders into separate files?

Reply:
Sure thing, I originally based this on the `DropShadowFilter` which defines the final blend pass shaders inline.